packages/app/src/runs/RunsEmpty.cy.tsx

```diff
@@ -21,6 +21,6 @@ describe('RunsEmpty', () => {
     } as any
 
     cy.contains(defaultMessages.specPage.banners.record.title).should('be.visible')
-    cy.contains('npx cypress run --component --record --key abcd-efg-1234').should('be.visible')
+    cy.get('[data-cy="terminal-prompt-input"]').should('have.value', 'npx cypress run --component --record --key abcd-efg-1234')
```

Comment:
Maybe this is a personal preference, but I like to use `contains` rather than test IDs because we are trying to act like the user in our tests. Users don't know about test IDs, they just scan the page for what they are trying to find, kind of like contains. 

Consider reverting your changes to these tests

Reply:
Thanks @astone123.
These changes were made based on some limitations noticed when using `.contain`.
As a result of updating command text, from DOM element to an input value as shown in the image below in `TerminalPrompt`

![Screenshot 2022-10-07 at 7 16 55 AM](https://user-images.githubusercontent.com/1501599/194481481-2f99ae6c-c823-4b2d-aea4-96ef6506fb72.png)

`cy.contain(<command text>)` fails, since it's not able to match command text anymore.
Alternatively, to bypass the usage of testId here, I could update this to `findByDisplayValue`.
WDYT? or perhaps there's an alternative way to keep the `.contain()` with current state of `TerminalPrompt`?

Reply:
Oh yeah I missed that. In this case, I think using `findByDisplayValue` would be the best approach. I can make that update and then we can move this through. Thanks!